README.md

```diff
@@ -1,7 +1,8 @@
-# features
+# Feature Tracking and Backlog
+
 Feature tracking repo for Kubernetes releases
 
-This repo only contains issues.  These issues are umbrellas for new features added to Kubernetes.  
+This repo only contains issues. These issues are umbrellas for new features to be added to Kubernetes. A feature can take multiple releases to complete. And a feature can be tracked as backlog items before work begins.
 
```

Comment:
@aronchick So, do you agree with saying:

## What is a feature

A feature is anything that:

- A blog post would be written about after its release (ex. [minikube](http://blog.kubernetes.io/2016/07/minikube-easily-run-kubernetes-locally.html), [PetSet](http://blog.kubernetes.io/2016/07/thousand-instances-of-cassandra-using-kubernetes-pet-set.html), [rkt container runtime](http://blog.kubernetes.io/2016/07/rktnetes-brings-rkt-container-engine-to-Kubernetes.html)
- Requires multiple parties/SIGs/owners participating to complete (ex. ???)

Do you have examples of multi-party features I can put in there?